tasks/time-series/time-series-forecasting/main.py

```diff
@@ -0,0 +1,118 @@
+# Imports
+import numpy as np
+import tensorflow as tf
+from sklearn.preprocessing import MinMaxScaler
+import keras
+from keras.models import Model
+from keras.layers import Dense, CuDNNLSTM, Input
+import datetime
+import os
+import shutil
+from numpy.random import seed
+from tensorflow import set_random_seed
+import matplotlib.pyplot as plt
+import argparse
+import mlflow.sklearn
+from mlflow import log_metric
+import mlflow.keras
+import warnings
+import sys
+
+# Initialize
+# Keras and Tensorflow Random Numbers
+seed(1)
+set_random_seed(2)
+
+# Clear
+# Previous Tensorflow Session
+tf.keras.backend.clear_session()
+
+# Parse Arguments
+parser = argparse.ArgumentParser()
+parser.add_argument("city_name", help="Name of the city to get climate data", type=str)
+args = parser.parse_args()
+
+# Define Variables
+lookback_window = 32
+steps_per_epoch = 364
+val_steps = 364 * 30
+LSTMunits = 100
+predict_steps = 364 * 10
+predict_shift = 0
+city = sys.argv[1]
+
+# Working directories
+model_filepath = 'Weather_Net_'+city+'.hf5'
+image_dir = 'images_prediction/'
+
+# Load Dataset - Daily climate data Hamburg 1951-01-01 - 2018-12-31
+# 0 Mess_Datum - 1 Wind_max - 2 Wind_mittel - 3 Niederschlagshoehe - 4 Sonnenstunden - 5 Schneehoehe - 6 Bedeckung_Stunden - 7 Luftdruck - 8 Temp_mittel - 9 Temp_max - 10 Temp_Min - 11 Temp_Boden - 12 Relative_Feuchte
+dataset = np.loadtxt(city+'.csv', dtype='float32', delimiter=';')
```

Comment:
Optional: Please keep only one example csv. With the amount of amazing contributions and potential of future, we are concerned about the size of the repository and trying to keep the size minimal. It would be great if you could add details on how to download files from the site on English version (so that usage can be increased). I tried but the view of the site gets change and I was unable to find download option easily.

Reply:
Done. Keeping only Hamburg as example. Added instruction for download, though I see a problem since some preprocessing/cleaning of the data is required. Maybe I can make a personal repo with more data csvs and crossreference to keep the main repo clean.

Reply:
@Doctor-Fu Optional: consider cleaning your data using the [anomaly detection](https://github.com/produvia/ai-platform/tree/master/tasks/methodology/anomaly-detection) task before applying your time-series forecasting task. You can accomplish this using MLflow [multistep workflows](https://www.mlflow.org/docs/latest/projects.html#building-multistep-workflows).